packages/server/graphql/mutations/helpers/notifications/notifyMattermost.ts

```diff
@@ -1,299 +0,0 @@
-import formatTime from 'parabol-client/utils/date/formatTime'
```

Comment:
This entire file is just dead code - it's not imported anywhere and none of the exports are used.